test/language.js

```diff
@@ -58,23 +58,23 @@ describe('language()', () => {
         expect(language).to.equal('');
     });
 
-    it('returns first preference if header has *', () => {
+    it('returns first preference if header has * and is unmatched', () => {
 
-        const language = Accept.language('da, en-GB, en, *', ['en-US']);
+        const language = Accept.language('da, en-GB, *', ['en-US']);
         expect(language).to.equal('en-US');
     });
 
     it('returns first found preference that header includes', () => {
 
         const language = Accept.language('da, en-GB, en', ['en-US', 'en-GB']);
-        expect(language).to.equal('en-GB');
+        expect(language).to.equal('en-US');
     });
 
-    it('returns preference with highest specificity', () => {
+    it('returns preference with highest order when equal weigths', () => {
 
         expect(Accept.language('da, en, en-GB', ['en', 'en-GB'])).to.equal('en');
         expect(Accept.language('da, en, en-GB', ['en-GB', 'en'])).to.equal('en-GB');
-        expect(Accept.language('en, en-GB, en-US')).to.equal('en-gb');
+        expect(Accept.language('en, en-GB, en-US')).to.equal('en');
```

Comment:
This changed due to removing the specificity clause. Now it just returns the first entry.